dinner_party_daniel.rb

```diff
@@ -0,0 +1,139 @@
+## Base Requirements
+# Create three arrays of beverages, food, and people.
+# Randomly assign a beverage and a food item to each person.
+# Print to the terminal a list of all of the people with their food and beverage ##along with a seat number.
+
+## Optional Enhancements
+# If you didn't already, only use one Hash variable which holds beverages, food, and people keys.
+# Allow the application to ask for user input to populate the three arrays. The user should be allowed to put in as many items as they want into each array. The program will need some way to determine when a user is finished adding values for a particular array.
+
+####################################################
+puts "Hey spouse, let's host a dinner party!"
+
+puts "Do you want to help plan it?"
+
+while true
+
+  answer = gets.chomp
+
+  if answer.downcase == "no" #run v1: hash pre-populated
+
+    puts "\nThat's OK, I can do it. Here is a guest list, with a food and drink for each person to bring:"
+
+    party = {
+      people: ["Bookis", "Elise", "Isaac", "Daniel", "Rachel",].shuffle,
+      food: ["cauliflower", "pickles", "Fig Newtons", "tortillas", "avocado"].shuffle,
+      drink: ["cider", "cashew milk", "broth", "sparkling water", "tea"].shuffle,
+      seat: ["5","4","3","2","1"]
+      ## Listing seats in backward order feels funny.
+      ### Bookis: try .each_with_index - could eliminate need for seat key altogether.
+      }
+
+    5.times do
+      puts """
+      Seat ##{party[:seat].pop}:\t#{party[:people].pop}
+      \t\tFood:#{party[:food].pop}; Beverage: #{party[:drink].pop}
+      """
+    end
+    break
+
+  elsif answer.downcase == "yes" #v2: collect user input
+
+    puts "Great! Who should we invite?"
+
+    party2 = {} #hash starts empty
+    people_array = [] #as do arrays for each hash value
+    food_array = []
+    drink_array = []
+    seat_array = ["5","4","3","2","1"] #pre-populated b/c we want seats ordered 1-5
+
+    #multiple user inputs into array, one array for each hash value
+
+    while true
+      person = gets.chomp
+      if person == ""
+        break
+      end
+      people_array << person
+    end
+
+    # puts "Test - is people_array still empty?"
+    # puts people_array.empty?
+    # puts
+
+    puts "What foods should we ask guests to bring?"
+
+    while true
+      food = gets.chomp
+      if food == ""
+        break
+      end
+      food_array << food
+    end
+
+    # puts "Test - is food_array still empty?"
+    # puts food_array.empty?
+    # puts
+
+    puts "What drinks should we ask guests to bring?"
+
+    while true
+      drink = gets.chomp
+      if drink == ""
+        break
+      end
+      drink_array << drink
+    end
+
+    # puts "Test - is drink_array still empty?"
+    # puts drink_array.empty?
```

Comment:
While learning, definitely not a big deal, but just fyi, comments that are old code of debug are generally deleted and only comments that are on the current code are generally left in the code. git can be used to refer to old code or debug. But only mentioning this for your future knowledge, for class it's totally fine.